### PR TITLE
Include all tags with exercises

### PR DESCRIPTION
**Pull request by nathanstitt**
Indicate which ones should be normally displayed via a new 'is_visible' property

Needs to be merged & deployed along with FE PR https://github.com/openstax/tutor-js/pull/990, otherwise teachers will see all tags on the HW builder.

### CODE REVIEW

File: app/representers/api/v1/exercise_representer.rb [hunk 1]
```diff
@@ -31,9 +31,6 @@ class ExerciseRepresenter < Roar::Decorator
     collection :tags,
                readable: true,
                writeable: false,
-               getter: ->(*) {
-                 (tags + tags.flat_map(&:teks_tags)).select{ |tag| tag.visible? }.uniq
-               },
```
Comment (jpslav) [inline app/representers/api/v1/exercise_representer.rb]: @nathanstitt do we still need the getter so we can include the `teks_tags`?

```ruby
getter: ->(*) { (tags + tags.flat_map(&:teks_tags)).uniq
```

?

Comment (nathanstitt) [inline app/representers/api/v1/exercise_representer.rb]: Yes, we do.  Thanks for pointing that out!

**Comment by jpslav** [comment]
Looks good.  @philschatz and @nathanstitt can merge this at their discretion (still says WIP so I won't merge)

**Comment by nathanstitt** [comment]
Sorry! forgot to remove the WIP prefix

**Pull request merged**